model-providers/ollama/runtime/src/test/java/io/quarkiverse/langchain4j/ollama/OllamaChatLanguageModelTest.java

```diff
@@ -0,0 +1,74 @@
+package io.quarkiverse.langchain4j.ollama;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+import java.util.HashMap;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import dev.langchain4j.agent.tool.ToolSpecification;
+import dev.langchain4j.data.message.UserMessage;
+import dev.langchain4j.model.chat.request.ChatRequest;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class OllamaChatLanguageModelTest {
```

Comment:
Can you please move the test to the `deploment` module? That's where `@QuarkusTest` tests belong